test/LEGO.AsyncAPI.Tests/Models/AsyncApiReference_Should.cs

```diff
@@ -426,6 +429,8 @@ public string Load(string reference)
                          userName:
                            description: User name.
                            type: string
+                     orderHistory:
+                       $ref: '#/components/OrderHistory'
```

Comment:
these component refs are wrong. 
its usually `#/components/{componentType}/{refId}`

Why its not resolving.

Reply:
Thanks @VisualBean I've updated with valid `componentType` object now I think. It's still not resolving though so I think the initial thought about internals not being resolved in externals is still valid.

Reply:
Ive debugged through it, and it seems to stem from the externalReferenceResolver.
Basically when the external reference is loaded, the subreferences are internal. So the ResolveReference<T> method simply returns null.
```
if (reference.IsExternal)
{
   ...
   return externalAsyncApiContent;
}

return null;
```

So - hopefully my referenceresolution followup PR will fix it.